deepinv/sampling/langevin.py

```diff
@@ -284,10 +56,10 @@ def __init__(
         self,
         prior,
         data_fidelity,
-        step_size=1.0,
-        sigma=0.05,
-        alpha=1.0,
-        max_iter=1e3,
+        step_size: float = 1.0,
+        sigma: float = 0.05,
+        alpha: float = 1.0,
+        max_iter: int = 1e3,
         thinning=5,
```

Comment:
I don't understand why for some parameters we enforce the type and for some other we don't. I think we should be consistent (or maybe I'm missing something). I'd suggest to enforce the type on the rest of the params, 
```suggestion
        thinning: int = 5,
```

Reply:
this should now be fixed